frappe/public/js/frappe/ui/messages.js

```diff
@@ -32,8 +32,16 @@ frappe.confirm = function(message, ifyes, ifno) {
 		],
 		primary_action_label: __("Yes"),
 		primary_action: function() {
-			if(ifyes) ifyes();
-			d.hide();
+			if(ifyes)
+			{
+				ifyes();
+				d.hide();
+			} 
+			else
+			{
+				ifno();
+				d.hide();
+			}
```

Comment:
```suggestion
			if (ifyes) ifyes();
			d.hide();
```

- Can you follow the coding style of frappe so that the code looks better.
- If everyone adds their own coding style to the code base, itll be a mess.